### PR TITLE
test: balances/manual and balances/prices test

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

**Review by kelsos — COMMENTED** [review]
I left some comments @lukicenturi

File: frontend/app/tests/unit/store/balances/manual.spec.ts [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+        bigNumberify(50).multipliedBy(1)
+      );
+    });
+  });
+
+  describe('addManualBalances', () => {
+    test('default', async () => {
+      await store.addManualBalance(balances[0]);
+
+      expect(useManualBalancesApi().addManualBalances([balances[0]]));
```
[the remark this answers is not in the document]
Comment (lukicenturi) [reply frontend/app/tests/unit/store/balances/manual.spec.ts:218]: my bad, should have called with `toHaveBeenCalledWith`😛

[the remark this answers is not in the document]
Comment (kelsos) [reply frontend/app/tests/unit/store/balances/manual.spec.ts]: It's ok, this is why we have reviews. It is easy to miss things sometimes 😄

**Review by kelsos — APPROVED** [review]
lgtm

**Pull request merged**